docs/guidance/map/connecting_publications.md

```diff
@@ -0,0 +1,429 @@
+```{workedexample} Connecting releases from different publications
+:tags: release
+```
+
+# Connecting releases from different publications
+
+Different OCDS publications can contain releases describing the same contracting process. In OCDS, each contracting or planning process is assigned a globally unique identifier, the [ocid](../../schema/identifiers.md#open-contracting-process-identifier-ocid). Ideally, releases describing the same contracting or planning process in different publications would reuse the same ocid so that users can [merge releases](../../schema/merging.md).
+
+This page describes the problems that can occur when reusing ocids across publications, how to connect releases from different publications without reusing ocids, and how to reuse ocids across publications.
+
+The examples on this page are based on Scotland's Public Contracts Scotland (PCS) publication and the UK's Find a Tender Service (FTS) publication, both of which include releases about the same contracting processes, covering the same contracting process stages and having many fields in common.
+
+## Problems with reusing ocids across publications
+
+Unless publications have disjoint coverage of OCDS fields or implement consistent mappings, reusing the ocids across publications can result in unpredictable and nonsensical results when releases are merged to create a compiled release. For example, unless publications use an identical approach to assigning [local identifiers](../../schema/identifiers.md#local-identifiers) to objects in arrays, then awards, contracts and other objects in arrays can overwrite and/or duplicate each other in nonsensical ways.
+
+### Example: Reusing ocids across Public Contracts Scotland and Find a Tender Service
+
+The PCS and FTS publications have many overlapping fields, but implement slightly different mappings. Therefore, if FTS were to reuse the ocids minted by PCS, merging releases would result in nonsensical data
```

Comment:
We've used "minted" in issues, but I think "assigned" is better for non-native speakers.

Reply:
Done in 1d2ac3fd1c7b08715083f49dcf4f68b3a9ec1dc5